ember-flight-icons/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hashicorp/ember-flight-icons",
-  "version": "0.0.1-beta",
+  "version": "0.0.2-beta",
```

Comment:
@amyrlam question: I don't usually bump version numbers (or at least, I don't do it until I am going to do a release). Should we be bumping patch versions with every PR?

Reply:
this bump was to test the changes from https://github.com/hashicorp/flight/pull/118#issuecomment-908574538, where we reworked the file structure

in this PR, we should commit the bumps since that's what's live in npm

definitely don't need to bump in the future unless there's an icon change, but to test that the instructions worked (https://github.com/hashicorp/flight/pull/138#discussion_r700513584) it made the most sense